packages/scandipwa/src/component/ProductActions/ProductActions.component.js

```diff
@@ -285,16 +286,12 @@ export class ProductActions extends PureComponent {
     renderCustomizableOptions() {
         const {
             product: {
-                options,
-                price_range: {
-                    minimum_price: {
-                        default_final_price_excl_tax: finalPrice = 0
-                    } = {}
-                } = {}
+                options
             } = {},
             getSelectedCustomizableOptions,
             productOptionsData,
-            device
+            device,
+            product: { type_id, price_range }
```

Comment:
You are already destructing the `product` object on the line 288. Please move these variables there.